R/utilities-writing-report.R

```diff
@@ -709,3 +709,53 @@ updateTableNumbers <- function(fileContent, pattern = "Table:", replacement = "T
   return(updateFigureNumbers(fileContent, pattern, replacement, anchorId))
 }
 
+#' @title copyReport
+#' @description Copy markdown report and its figures (using their paths)
+#' @param from path of initial .md file to copy
+#' @param to path of destination .md file to be copied
+#' @param keep logical defining if initial .md file and figures are kept
+#' @export
+copyReport <- function(from, to, keep = FALSE) {
+  validateIsFileExtension(from, "md")
+  validateIsFileExtension(to, "md")
+  validateFileExists(from)
+  validateIsLogical(keep)
+  # If from is 
+  if(from == to){
+    return(invisible())
+  }
```

Comment:
dangerous, same path could be written in the different ways (especially under windows).
Better e.g. something like this:
```
if(tolower(normalizePath(paste0(from, "/"))) == tolower(normalizePath(paste0(to, "/")))){
...
```

MAybe there is more elegant way to do it in R, don't know.

Reply:
That's true.
Unfortunately, I don't think there is more elegant than that